content/docs/command-reference/exp/remove.md

```diff
@@ -25,6 +25,8 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `--queue` - remove all experiments that haven't been run yet (defined via
   `dvc exp run --queue`).
 
+- `-A`, `--all` - remove all experiments in this project.
```

Comment:
Why do we specify "in this project"? Is there a way to `exp remove` from remotes?

Reply:
Also, does it include queued ones? If so
```suggestion
- `-A`, `--all` - remove all experiments (includes `--queue`).
```

Reply:
@jorgeorpinel 
Not included for now.

Reply:
> Why do we specify "in this project"? Is there a way to exp remove from remotes? Otherwise

Yes there is one, https://github.com/iterative/dvc/pull/6471.

Reply:
> Yes there is one, iterative/dvc#6471.

OK but that's not merged yet so unrelated to this PR 🙂 so let's not make the distinction yet but we'll revisit very soon per iterative/dvc#6471 😬 . Committing 2nd suggestion (resolving).